.github/workflows/app-build-verify.yml

```diff
@@ -186,6 +186,53 @@ jobs:
         briefcase build windows VisualStudio
         briefcase package windows VisualStudio --adhoc-sign
 
+    - name: Build Linux System project (Ubuntu, local)
+      if: |
+        startsWith(inputs.runner-os, 'ubuntu')
+        && startsWith(inputs.python-version, '3.10')
```

Comment:
This check should probably be more robust. The underlying purpose of the check is to ensure this step only runs if requested python is the system Python for the Ubuntu version being tested against; however, we don't actually know the version of Ubuntu being tested since it may well be `ubuntu-latest` and that will not always ever resolve to a single version across all linux runners.

A potential solution: (note: untested)
```
fromJSON('{"ubuntu-22.04": "3.10", "ubuntu-20.04": "3.8"}')[inputs.runner-os] == 'inputs.python-version'
```

What's probably a better approach, though, is an earlier step that provides the system Python of the runner's OS as an output that can be checked here.

A different type of approach is to simply skip verifying the Python version here at all and instead require all callers to specify a proper Python version else this fails. This would also mean the caller should be specifying a specific ubuntu version to be in line with my other comment.

open to thoughts ofc.

Reply:
Checking the system python version seems like a more viable option. I'll take a swing at getting that to work.